contracts/ICO_controller.sol

```diff
@@ -239,35 +197,27 @@ contract ICO_controller is Ownable, TransferableInterface {
             escrowIco.transfer(this.balance.div(2));
             // send other 50% to multisig holder address
             holder.transfer(this.balance);
-
-            // increase airdrop supply with unspent pull amounts
-            airdropSupply = airdropSupply.add(PULL_SUPPLY).sub(pullSpentSupply);
         }
         crowdsaleFinished = true;
 
     }
 
     function finishCrowdsaleBurnUnused() external onlyOwner {
-        require(crowdsaleFinished);
+        // require(crowdsaleFinished == true);
```

Comment:
Why is this `require` commented out? Do we want to allow the burning before the crowdsale finished?